Content.IntegrationTests/Tests/VendingMachineRestockTest.cs

```diff
@@ -13,346 +13,362 @@
 using Content.Shared.VendingMachines;
 using Content.Shared.Wires;
 using Content.Server.Wires;
+using Content.Shared.Broke;
+using Content.Shared.DispenseOnHit;
+using Content.Shared.VendingMachines.Components;
 
-namespace Content.IntegrationTests.Tests
+namespace Content.IntegrationTests.Tests;
+
+[TestFixture]
+[TestOf(typeof(VendingMachineRestockComponent))]
+[TestOf(typeof(VendingMachineSystem))]
+public sealed class VendingMachineRestockTest : EntitySystem
 {
-    [TestFixture]
-    [TestOf(typeof(VendingMachineRestockComponent))]
-    [TestOf(typeof(VendingMachineSystem))]
-    public sealed class VendingMachineRestockTest : EntitySystem
-    {
-        private const string Prototypes = @"
+    private const string Prototypes = @"
 - type: entity
-  name: HumanDummy
-  id: HumanDummy
-  components:
-  - type: Hands
-  - type: Body
-    prototype: Human
+name: HumanDummy
+id: HumanDummy
+components:
+- type: Hands
+- type: Body
+prototype: Human
```

Comment:
Why remove yaml indentation?

Reply:
> Why remove yaml indentation?

Hi. How can I also select the code from a specific script and send it back to you?
Since, unfortunately, I do not know this yet, then in the script VendingMachineRestockTest.cs, from lines 19 to 20, I had to remove the curly brackets that covered the entire script. Because of this, the code moved one tab to the left.

Reply:
> Why remove yaml indentation?

Returned it back